dev/drivers/scripts/stats/cam/jevs_cam_href_spcoutlook_stats.sh

```diff
@@ -33,7 +33,7 @@ export vhr=00
 export run_mpi=yes
 export gather=yes
 
-export COMIN=/lfs/h2/emc/vpppg/noscrub/$USER/$NET/$evs_ver_2d
+export COMIN=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/$NET/$evs_ver_2d
```

Comment:
revert lines 14 and 36 to previous defitions.

Reply:
Done.